nobl9/resource_role_binding.go

```diff
@@ -94,6 +93,21 @@ func unmarshalRoleBinding(d *schema.ResourceData, objects []n9api.AnyJSONObj) di
 	return diags
 }
 
+// getRoleBindingByType returns role binding depending on type that was applied
+//
+// Role bindings have two types, organization role and project role, both can have same name.
+// If api return to us more than one role binding (when there are two roles bindings with same name)
+// we specify which one should be processed by checking if project is not empty.
+func getRoleBindingByType(d *schema.ResourceData, objects []n9api.AnyJSONObj) n9api.AnyJSONObj {
+	if len(objects) == 1 {
```

Comment:
Let's always check the role's type to be 100% sure that this is a proper one. That will be beneficial when a project role is deleted directly from the UI.

Reply:
Right